Software/dist_windows/script32.iss

```diff
@@ -57,18 +57,13 @@ Source: "content/Qt5Network.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5SerialPort.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5Widgets.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "gpl-3.0.txt"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icudt54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuin54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuuc54.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
windeployqt says the icu* dlls are not needed? Last time I tried Qt5Core had a hard dependency on them